content/faculty/structure.md

```diff
@@ -0,0 +1,88 @@
+title: Оргструктура факультета
+slug: structure
+category: faculty
+tags: [_nav]
+submenu_index: 4
+---
+
+<div class="bg-structure">
```

Comment:
Add `type: org_structure` to meta data and move the contents of the page into `templates/org_structure`.